code/datums/move_intent/move_intent.dm

```diff
@@ -23,7 +23,7 @@
 		return TRUE
 
 /decl/move_intent/run/can_be_used_by(var/mob/user)
-	if(user.incapacitated(INCAPACITATION_ALL))
```

Comment:
overly specific code like this shouldn't be used in general places

That aside, there's no reason for stasis to prevent sprinting. As long as its modifying movespeed, people can sprint in slow motion all they want